### ISSUE TITLE
Add support Alamofire 4.5.

### ISSUE THREAD

**Issue by buh** [ciscospark/spark-ios-sdk#33]
Hi guys,

I have a conflict in dependencies for Alamofire framework. Spark-iOS SDK locks Alamofire version to 4.0.x, but I tried to update Podfile like below and it compiles without any errors.
Could you update your pod spec to support Alamofire for 4.5?
Thanks!
```
	pod 'Alamofire', '~> 4.0'
	pod 'ObjectMapper', '~> 2.0'
	pod 'AlamofireObjectMapper', '~> 4.0'
```

**Comment by rikvdbrule** [comment]
I created PR #34 that updates the `Podfile` and `SparkSDK.podspec` that locks on the major version of the pods. An alternative would be to keep the lock on the patch version as it is now but to increment the minor version in the files.

**Comment by rikvdbrule** [comment]
Could someone please take a look at this issue and the accompanying PR? We would love to get this merged into `develop`, it's blocking us from moving away from our custom fork of spark-ios-sdk.

**Comment by Noble-Six** [comment]
Hi @rikvdbrule, merged PR#33 into develop branch.Please use this branch to test your project.Contact us if have any question.
